mup-placeholder.js

```diff
@@ -20,7 +20,8 @@ module.exports = {
     env: {
       ROOT_URL: 'https://apm.YOUR_DOMAIN.com',
       MONGO_URL: 'mongodb://mongodb/meteor',
-      MONGO_OPLOG_URL: 'mongodb://mongodb/local'
+      MONGO_OPLOG_URL: 'mongodb://mongodb/local',
+      MAIL_URL: ''
```

Comment:
Do you know if mailing works?

Reply:
Yes, mailing works (I tested invinting users to an application)